tests/browser/http/test_content_disposition.py

```diff
@@ -20,11 +20,11 @@
 """Tests for qutebrowser.browser.http.parse_content_disposition."""
 
 import os
-import unittest
 import logging
 
```

Comment:
Nitpicking is always appreciated! :smile:

Imports are grouped (according to [PEP8](https://www.python.org/dev/peps/pep-0008/#imports)) into stdlib/3rdparty/local imports, so that's fine.